lib/IssuanceInitiation.ts

```diff
@@ -0,0 +1,28 @@
+import { convertJsonToURI, convertURIToJsonObject } from './functions';
+import { IssuanceInitiationRequestPayload, IssuanceInitiationWithBaseUrl } from './types';
+
+export class IssuanceInitiation {
```

Comment:
It seems more like a Util perhaps it should be renamed and similar to HTTP Utils. And perhaps it can go in that folder. 

Also functions seems incorrect when it contains the classes ...

Reply:
Nah I don't want to call everything a meaningless Util. That literally is saying nothing

This is a core class for interaction of the developer. As you can see it is using functions with some params, so that we do not have to ask the developer to know with which arguments the function should be called to begin with. Why would that go into a Utils folder?

Reply:
There is a directory named functions. If this class were to move there ... we would need to rename that directory. But since we are not moving I think that name is fine.